molgenis-data-vcf/src/test/java/org/molgenis/data/vcf/importer/VcfImporterServiceTest.java

```diff
@@ -121,7 +122,7 @@ public void forEachBatched(Consumer<List<Entity>> consumer, int batchSize)
 		when(source.getRepository(entityTypeId0)).thenReturn(repo0);
 		String importPackageId = "package";
 		Package importPackage = mock(Package.class);
-		when(metaDataService.getPackage(importPackageId)).thenReturn(importPackage);
+		when(metaDataService.getPackage(importPackageId)).thenReturn(of(importPackage));
```

Comment:
In this case a static import is less readable in my opinion, because `of` is a common factory method name.